src/hooks/tests/use-archive-site.test.ts

```diff
@@ -0,0 +1,92 @@
+import { renderHook, act } from '@testing-library/react';
```

Comment:
We could add the instructions about how to execute this test.

```suggestion
// To run tests, execute `npm run test -- src/hooks/tests/use-archive-site.test.ts` from the root directory
import { renderHook, act } from '@testing-library/react';
```

Reply:
What is the reason to add instruction in code for this particular text?